site/content/docs/traces-open-telemetry/exporting-traces/grafana-tempo.md

```diff
@@ -13,14 +13,34 @@ beta: true
 aliases:
   - "/docs/open-telemetry/exporting-traces/grafana-tempo/"
 ---
+You can connect the Traces export to Grafana using the OpenTelemetry Integration and Grafana Tempo.
 
-1. Make sure you have Grafana Tempo installed, running and accessible. If you are running Grafana Cloud, the Tempo
-   datasource is already pre-configured. Find it at **Connections** > **Datasources** > **Tempo**.
-2. After installing, grab the Tempo endpoint URL. It should look like `https://tempo-eu-west-0.grafana.net/tempo`.
-3. Under the section **Authentication**, grab the user and password for the Tempo endpoint.
-4. As Tempo uses Basic Authentication, you need to provide the user and password in a `Authorization: Basic user:password` HTTP header in
-   Checkly integration settings, where the `user:password` section is base64 encoded. You can use an online tool like [base64encode.net](https://www.base64encode.net/)
-   to encode your user and password.
+1. Make sure that you have a Grafana Tempo instance running on Grafana Cloud. (It comes with one by default.)
+   
+2. In Grafana Cloud, head over to **Connections** > **Add new connection** > **OpenTelemetry (OTLP)**.
+  ![Add an OTLP connection in Grafana Cloud](/docs/images/otel/export-traces/grafana-cloud-opentelemetry.png)
 
-Find all the details in the [Grafana Tempo OpenTelemetry documentation](https://grafana.com/docs/tempo/latest/).
+3. Copy the endpoint URL and append `v1/traces` to it.
```

Comment:
Out of curiosity: In our account https://checklyhq.grafana.net/connections/add-new-connection/open-telemetry the configuration looks very different - where are you setting this up?

Reply:
Aha, this is the trick, I hope I can clarify better in the docs from this. 

that one is the Grafana Instance for your Stack at orgName.grafana.net/connections but there's also Grafana Cloud at grafana.com/orgs/orgName, https://grafana.com/orgs/checklyhq/ 

Does that make sense?